packages/owl-bot/test/copy-code.ts

```diff
@@ -115,9 +119,13 @@ describe('copyDirs', () => {
           source:
             '/google/cloud/asset/.*/.*-java/(grpc-google-cloud-asset-.*)-java',
           dest: '/$1',
-          'rm-dest': '/grpc-google-cloud-asset-.*',
         },
       ],
+      'deep-remove-regex': ['/grpc-google-cloud-asset-.*'],
+      'deep-preserve-regex': [
```

Comment:
Were we able to exercise the new logic fully with existing tests? Wondering if we need any additional test cases.

Reply:
Lines 107 and 108 exercise the new logic, and lines 137 and 146 confirm it behaves as expected.